### DISCUSSION TITLE
Suggestion: add new energy storage

### DISCUSSION THREAD

**Discussion by Uran-228** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1648] [General]
As for me, energy storage is an important part of the game, and for my survival situation, lithium-ion batteries are difficult to manufacture, and conventional energy storage is unprofitable. Therefore, I suggest adding sodium-ion energy storage facilities.
Storage description
I think that the volume of energy can be 125-200 MHE maybe
And the craft can look like that:
12 PVC, 16 cobalt powder, 16 sodium
To make an energy storage in an assembly machine, the production time is 7 seconds

Thanks for reading this suggestion :-)
If you like it I can offer another idea 😉
GOOD LUCK! 😎

**Comment by Uran-228** [comment]
As a bonus, sodium can be more useful in mod.

**Comment by HollyEreTic** [comment]
Agree with you!
